### PR TITLE
User can regulate the quanitity of food items from menu page itself

### PR DESCRIPTION
**Pull request by sujal1256**
Fixes: #24 
- User can change the number if items from menu page itself
- Quantity of Cart items are stored in redux
- Changed addToCart and removeFromCart function in cartSlice
- Made another function "removeItemFromCart" to bring that quantity to 0

Checkout the changes below to see the added functionality

https://github.com/user-attachments/assets/35a349b5-b72d-45b1-81e9-e38be5b5767e

### CODE REVIEW

**Comment by sujal1256** [comment]
Hey, @mohitahlawat2001 why these checks have failed I have approved the deployment on vercel, still the check haven't passed yet.

File: src/components/Cart.js [hunk 1]
```diff
@@ -1,115 +1,136 @@
...
-
-    const [quantities, setQuantities] = useState(
-        cart.reduce((acc, item) => {
-            acc[item?.card?.info?.id] = 1;
-            return acc;
-        }, {})
+  const cart = useSelector((state) => state.cart.items);
+  const dispatch = useDispatch();
+
+//   No need of quantities array, every thing is stored in redux store ( cart slice)
```
Comment (mohitahlawat2001) [inline src/components/Cart.js:9]: remove the comments

File: src/components/Cart.js [hunk 2]
```diff
@@ -1,115 +1,136 @@
...
+    dispatch(clearCart());
+  };
+
+  const handleRemoveFromCart = (item) => {
+    dispatch(removeFromCart(item));
+  };
+
+//   No need of this function every thing is done using cartslice functions
+
+//   const handleQuantityChange = (item, operation) => {
```
Comment (mohitahlawat2001) [inline src/components/Cart.js:28]: remove comments here also

**Review by mohitahlawat2001 — CHANGES_REQUESTED** [review]

**Comment by mohitahlawat2001** [comment]
remove this comments in code on which you work on only leave documentation comments, i will merge this , make sure you star ⭐  repo and follow on github

**Comment by sujal1256** [comment]
I have removed the comments

**Pull request merged**